index.bs

```diff
@@ -331,7 +331,7 @@ Each {{WebSocket}} object has an associated <dfn>url</dfn> (a [=URL record=]).
  1. If |urlRecord|'s [=fragment=] is non-null, then throw a "{{SyntaxError}}" {{DOMException}}.
  1. If |protocols| is a string, set |protocols| to a sequence consisting of just that string.
  1. If any of the values in |protocols| occur more than once or otherwise fail to match the
-    requirements for elements that comprise the value of <a dfn><code>Sec-WebSocket-Protocol</code></a>
+    requirements for elements that comprise the value of <a dfn>`Sec-WebSocket-Protocol`</a>
```

Comment:
This should `<a http-header>` (might require definition updates) and I believe the backticks are usually outside.